nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Migrating the Distributor to use Sender Side Distribution
+tags:
+ - upgrade
+ - migration
+related:
+ - samples/scaleout/distributor-upgrade
+---
+
+The [distributor](/nservicebus/scalability-and-ha/distributor) has been deprecated and replaced by [sender-side distribution](/nservicebus/msmq/scalability-and-ha/sender-side-distribution.md) with NServiceBus Version 6.
+
+## Client-side distribution
+
+With sender-side distribution, senders are aware of scaled out receivers, allowing senders to distribute messages directly to the receivers instead of using an intermediary distributor.
+
+
+## Upgrade scenario
+
+This process aims to allow upgrade in production environments without missing events and minimal downtime.
+
+* Upgrade all endpoints that interact with the distributor to Version 6 first. For example, upgrade all endpoints that send command messages to the Distributor endpoint for the load to be distributed, or endpoints that send subscription messages to the distributor, subscribing to events when a worker node publishes an event. At this stage, do not upgrade the workers to Version 6.
+* Apply the following steps for each scaled out instance enlisted to the Distributor, one instance after another:
```

Comment:
>Apply the following steps for each scaled out instance enlisted to the Distributor, one instance after another:

@timbussmann - I think rather than `scaled out instance`, people who use the distributor understand the term worker better. Did you mean worker node here?